arangod/IResearch/IResearchView.cpp

```diff
@@ -1043,28 +1043,17 @@ Result IResearchView::updateProperties(
     // ...........................................................................
 
     std::unordered_set<DataSourceId> collections;
-
-    if (partialUpdate) {
-      mtx.unlock(); // release lock
-
-      auto lock = irs::make_lock_guard(_updateLinksLock);
-
-      return IResearchLinkHelper::updateLinks(
-        collections, *this, links, getDefaultVersion(isUserRequest));
```

Comment:
This call is exactly the same as the second one, it just uses an empty default value.